src_ext/Makefile

```diff
@@ -70,7 +70,7 @@ endef
 
 %.stamp: %.download
 	mkdir -p tmp
-	cd tmp && $(if $(patsubst %.tar.gz,,$(URL_$*)),bunzip2,gunzip) -c ../$(notdir $(URL_$*)) | tar x
+	cd tmp && tar x$(if $(patsubst %.tar.gz,,$(URL_$*)),j,z)f ../$(notdir $(URL_$*))
```

Comment:
This is a revert of e740dd6b8880462c5b581a6b404917451611cb47 and can't go through as-is: see https://github.com/ocaml/opam/issues/2158

I'm being lazy and not booting my FreeBSD VM, but is it really possible to have `tar -j` working yet no `bunzip2`? Would it be better to use `-d` with the `bzip2` or `gzip` commands instead?